crates/forge/tests/cli/test_cmd.rs

```diff
@@ -704,9 +738,22 @@ contract CounterTest is Test {
 
     // make sure there are only 61 runs (with proptest shrinking same test results in 298 runs)
     cmd.args(["test"]).assert_failure().stdout_eq(str![[r#"
-...
-[..]testAddOne(uint256) (runs: 61, μ: [..], ~: [..])
-...
+Compiling 23 files with [SOLC_VERSION]
+[SOLC_VERSION] [ELAPSED]
+Compiler run successful!
+
+Ran 1 test for test/CounterFuzz.t.sol:CounterTest
+[FAIL. Reason: panic: arithmetic underflow or overflow (0x11); counterexample: calldata=0xa76d58f5ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff args=[115792089237316195423570985008687907853269984665640564039457584007913129639935 [1.157e77]]] testAddOne(uint256) ([RUNS])
```

Comment:
this test expect for runs to be 61, let's not redact [RUNS] here but maybe keep the short version of `testAddOne(uint256) (runs: 61, μ: [..], ~: [..])` which clearly indicates what's tested?

Reply:
oh, right, let's not redact them at all